manifest.toml

```
@@ -0,0 +1,72 @@
+packaging_format = 2
+
+id = "mantis"
+name = "Mantis"
+description.en = "Bug tracking system"
+description.fr = "Système de suivi des bugs"
+
+version = "2.25.8~ynh1"
+
+maintainers = ["eric_G"]
+
+[upstream]
+license = "GPL-3.0-only"
+website = "https://mantisbt.org/"
+demo = "https://mantisbt.org/bugs/my_view_page.php"
+admindoc = "https://mantisbt.org/documentation.php"
+code = "https://github.com/mantisbt/mantisbt/"
+
+[integration]
+yunohost = ">= 11.2"
+architectures = "all"
+multi_instance = true
+ldap = true
+sso = false
+disk = "50M"
+ram.build = "50M"
+ram.runtime = "50M"
+
+[install]
+    [install.domain]
+    type = "domain"
+
+    [install.path]
+    type = "path"
+    default = "/mantis"
+
+    [install.admin]
+    type = "user"
+
+    [install.init_main_permission]
+    type = "group"
+    default = "visitors"
+
+    [install.default_access_level]
+    ask.en = "Choose the default access level for new users"
+    ask.fr = "Choisissez le niveau d'accès par défaut pour les nouveaux utilisateurs"
+    type = "select"
+    choices = ["VIEWER", "REPORTER", "UPDATER", "DEVELOPER", "MANAGER"]
+    default = "REPORTER"
+
+[resources]
+
+    [resources.sources]
+        [resources.sources.main]
+        url = "https://sourceforge.net/projects/mantisbt/files/mantis-stable/2.25.8/mantisbt-2.25.8.zip/download"
+        sha256 = "c8bd4b585183cb42cdd9fbb00c4e5d5945e01533882d7e534b33dc45025752ff"
```

```suggestion
        url = "https://github.com/mantisbt/mantisbt/archive/refs/tags/release-2.25.8.tar.gz"
        sha256 = "8d96bf93a283267fc088229661432831829d18b9fc1d20bff35df64be69ea27d"
```

zzz sourceforge download urls are completely stupid